go.mod

```diff
@@ -23,7 +23,9 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/jaegertracing/jaeger v1.22.0
+	github.com/knadh/koanf v1.0.0
```

Comment:
I don't see Viper being removed, is it something we should see?

Reply:
Not until the deprecated unmarshaller path is removed: https://github.com/open-telemetry/opentelemetry-collector/blob/df76aa36cd124ba4ae1b3d1ac1941155f86ea568/config/configloader/config.go#L537